cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -1598,17 +1629,11 @@ def _clear(*_, file_path: str, file_ctime: str):
             processes_finished_correctly = True
 
         self.assertTrue(processes_finished_correctly)
+        self.assertGreater(os.path.getmtime(first_export_path), initial_file_modfication_time)
 
         # terminate() may break the locks, don't try to acquire
         # https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process.terminate
         self.assertTrue(export_checked_the_file.get())
-        self.assertTrue(export_created_the_file.get())
-
-        self.assertFalse(clear_removed_the_file.get())
-
-        new_export_path = export_file_path.get()
-        self.assertGreater(len(new_export_path), 0)
-        self.assertTrue(osp.isfile(new_export_path))
```

Comment:
Maybe checks for the clear process activities should be restored? I'd expect it to check the exported file.

Reply:
I do not understand why these checks should be restored.
> self.assertFalse(clear_removed_the_file.get())

Custom _clear function checks that `mock_os_remove` is not called

> self.assertTrue(export_created_the_file.get())

export_created_the_file is not set anymore since we do not create a file, only update last modification date

> new_export_path = export_file_path.get()
> self.assertGreater(len(new_export_path), 0)
> self.assertTrue(osp.isfile(new_export_path))

we check that the initial export creates a file and then the file modification date is changed

Reply:
I don't insist on restoring the same checks that were there before the PR changes, but, as I said, it would be nice to check that all the conditions supposed by the test were actually used. The implementation of the test callbacks and of the functions being tested can change, and we just want to make sure these tests can spot a problem.

Reply:
:crying_cat_face: